sandbox/OpenIddict.Sandbox.Uno.Client/OpenIddictClientCryptographicKeys.cs

```diff
@@ -39,6 +41,7 @@ internal static RsaSecurityKey GetRsaCngKey(string name, CngKeyUsages usages, Cn
 
         return new RsaSecurityKey(myRsa);
 #else
+        System.Security.Cryptography.CngKey key;
         #pragma warning disable CA2000 // Dispose objects before losing scope
                 if (CngKey.Exists(name, provider, CngKeyOpenOptions.UserKey))
                 {
```

Comment:
Ident below as standard C#
```suggestion
#pragma warning disable CA2000 // Dispose objects before losing scope
        if (CngKey.Exists(name, provider, CngKeyOpenOptions.UserKey))
        {
```